backend/app/services/chat/trigger/core.py

```
@@ -505,3 +553,665 @@ async def _stream_chat_response(
+            task_id=task_id,
+            subtask_id=subtask_id,
+            error=str(e),
+        )
+
+    finally:
+        # Unregister stream to clean up local event and Redis cancel flag
+        await session_manager.unregister_stream(subtask_id)
+
+
+async def _stream_with_bridge(
+    stream_data: StreamTaskData,
```

Ruff (0.14.10)

Unused function argument: `stream_data`

(ARG001)